scripts/TestHarness/testers/RavenFramework.py

```diff
@@ -168,7 +168,7 @@ def check_runnable(self):
     ## required module is present, but too old
     if _notQAModules and _checkVersions:
       self.set_fail('skipped (Incorrectly versioned python modules: ' +
-                    " ".join(['{}-{}!={}'.format(*m) for m in _notQAModules]) +
+                    " ".join(['{}-{}'.format(*m) for m in _notQAModules]) +
```

Comment:
This reverts useful changes.  Can you double check that you didn't accidentally do this elsewhere in the pull request?

Reply:
Sorry. Reverted.